src/collector.ts

```diff
@@ -132,6 +157,17 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        function getReplaceDependency(dep: IDependency): IDependency {
```

Comment:
This function need to be moved out of the parent function. Also name should be changed. e..g applyReplaceMap